spec/std/record_spec.cr

```diff
@@ -19,6 +19,12 @@ end
 
 private record Sub < Base, x : Int32
 
+private record CustomInitializer, id : Int32, active : Bool = false do
```

Comment:
What about just `record Foo, active = false`.

Reply:
Wanted to replicate what was happening within the original issue; which would be using a custom override, setting one value, but leaving the other to be `false`.

Reply:
Maybe I should have been more clear, but what I was asking for was a spec that

```cr
private record CustomInitializer, id : Int32, active = false
```

works, as well as the record you have above.